src/plugin.tsx

```diff
@@ -37,46 +34,47 @@ export const rootRouteRef = createRouteRef({
   title: 'Firebase functions list',
 });
 
-export const plugin = createPluginAny({
+export const plugin = createPlugin({
   id: 'firebase-functions',
-  // comment-out 'apis' property for local standalone development (until backstage-core is upgraded to support 'apis' in type definition)
   apis: [
-    createApiFactoryAny(firebaseFunctionsApiRef, new FirebaseFunctionsClient()),
+    createApiFactory(firebaseFunctionsApiRef, new FirebaseFunctionsClient()),
```

Comment:
This is the part which uses the new plugin API that Spotify created right?

Reply:
Yes. Because it's inside of the backstage repo it's taking this function straight from the backstage/core source. If you would run this as standalone, it would take the function from the npm package (which right now is not up-to-date)